opencti-platform/opencti-graphql/src/database/data-initialization.js

```diff
@@ -247,12 +247,21 @@ const createBasicRolesAndCapabilities = async (context) => {
     description: 'Default role associated to the default group',
     capabilities: [KNOWLEDGE_CAPABILITY],
   });
-  await addRole(context, SYSTEM_USER, {
+
+  let administratorRoleInput = {
     name: ROLE_ADMINISTRATOR,
     description: 'Administrator role that bypass every capabilities',
     capabilities: [BYPASS],
-  });
-  const connectorRole = await addRole(context, SYSTEM_USER, {
+  };
+  if(isFeatureEnabled((PROTECT_SENSITIVE_CHANGES_FF))){
+    administratorRoleInput = {
+      ...administratorRoleInput,
+      is_sensitive_changes_allow: false
```

Comment:
false ?

Reply:
need to check if CSM or platform use this group too

Reply:
done